test/test_core.py

```diff
@@ -2434,6 +2434,18 @@ def test_aborting_new(self, args):
     self.emcc_args += args
     self.do_core_test('test_aborting_new.cpp')
 
+  @parameterized({
+    'nogrow': (['-sABORTING_MALLOC=0'],),
+    'grow': (['-sALLOW_MEMORY_GROWTH', '-sMAXIMUM_MEMORY=18MB'],)
```

Comment:
Why isn't `-sABORTING_MALLOC=0` needed in both cases here?

Reply:
If I understand #11131 correctly, `ABORTING_MALLOC` is implicitly disabled when `ALLOW_MEMORY_GROWTH` is enabled.

Reply:
Ah that make sense.  I wonder if a comment there might make sense.. or just add `-sABORTING_MALLOC=0` for clarity anyway?